src/countries/dto/deleteCountryRequest.ts

```diff
@@ -0,0 +1,6 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class deleteCountryRequest {
+  @ApiProperty()
+  id: number;
```

Comment:
it's good to use `@IsNumberString({ message: 'Id should be a number ' })` bi hk matarih